tools/doc-generator/main.go

```diff
@@ -93,6 +95,11 @@ var (
 			structType: reflect.TypeOf(storage.Config{}),
 			desc:       "The storage_config configures where Cortex stores the data (chunks storage engine).",
 		},
+		{
+			name:       "flusher_config",
+			structType: reflect.TypeOf(flusher.Config{}),
+			desc:       "The flusher_config configures how flusher to flush all the chunks.",
```

Comment:
@codesome Is my suggested change correct?

```suggestion
			desc:       "The flusher_config configures the WAL flusher target, used to manually run one-time flushes when scaling down ingesters.",
```